gym_roboy/envs/ros_proxy.py

```diff
@@ -128,13 +130,14 @@ def _publish_new_goal_on_rviz(self, goal_joint_angle):
         self.sphere_axis2.publish(msg2)
 
     def get_new_goal_joint_angles(self):
+        #self.node.get_logger().info("Reached goal joint angles: " + str(self.read_state().joint_angle))
```

Comment:
Lets get rid of commented out code

Reply:
I need it for debugging how close we are to the goal joint angle. It's good for deciding the number in above review.